Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -180,17 +197,24 @@ public class ArcGISARView: UIView {
     /// - Parameter initialTransformation: The initial transformation for originCamera offset.
     /// - Returns: Whether setting the `initialTransformation` succeeded or failed.
     public func setInitialTransformation(initialTransformation: AGSTransformationMatrix) -> Bool {
```

Comment:
I'm guessing the return value is for future use? Normally I would suggest that failure operation should throw, but that might be a bit heavy handed in this case. I would suggest that you annotate the method with `@discardableResult` as a convenience for callers who don't care whether the operation succeeded. Same with the other `setInitialTransformation` methods.

Reply:
I think the other method, which *can* return `false` needs the non-discardable result.  The reason this one has a return value of bool is that it is an "overloaded" method, so in the design the signature was the same except for the argument.

I was going back-and-forth debating whether to just remove the Bool return value, since it's always going to be true (and it seems to compile fine with it removed).

Thoughts?

Reply:
I see what you mean. Part of me thinks that the return value isn't necessary for any of the functions since the user can check the value of `initialTransformation` to see if it has changed. But that's up to you. I don't see a big problem with different functions in the same family having different return types.

Reply:
The `Bool` return value is necessary for the method that takes a screenPoint as the hit test the operation is based on may return `nil`.  I'm going to leave the `Bool` return for the other `setInitialTransformation` method as that is what the design states, but I will add the `@discardableResult` keyword.